README.md

```diff
@@ -16,8 +16,8 @@
   <a title="Follow on Twitter" href="https://twitter.com/Nodejs">
     <img src="https://img.shields.io/twitter/follow/Nodejs.svg?style=social&label=Follow%20@Nodejs">
   </a>
-  <br />
-  <br />
+  <br/>
```

Comment:
These should be undone, there should be a space before the self closing tags

Reply:
Done @nschonni . 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good